config/locales/models/en.yml

```diff
@@ -145,6 +148,7 @@ en:
               changed_too_recently:
                 one: can only be changed once per day. You last changed your user name on %{renamed_at}.
                 other: can only be changed once every %{count} days. You last changed your user name on %{renamed_at}.
+              taken: has already been taken
```

Comment:
This is already the [default text](https://github.com/svenfuchs/rails-i18n/blob/556eb65a4b701eec430cfc5954f103a102ca1ae5/rails/locale/en.yml#L133) for the `taken` error, so I don't think this is necessary.